ckan/migration/versions/102_ff13667243ed_create_conditinal_index_on_user.py

```diff
@@ -0,0 +1,24 @@
+"""create conditinal index on user
+
+Revision ID: ff13667243ed
+Revises: d111f446733b
+Create Date: 2022-06-22 11:38:27.553446
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'ff13667243ed'
+down_revision = 'd111f446733b'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_index("idx_only_one_active_email", "user", ["email", "state"], unique=True, postgresql_where=sa.text('"user".state=\'active\''))
+
+
+def downgrade():
+    op.drop_index("only_one_active_email")
```

Comment:
Shouldn't it be 
```
op.drop_index("idx_only_one_active_email")
```
, with the `idx_` prefix?

Reply:
already fixed in https://github.com/ckan/ckan/pull/6916/commits/1d447fddb738eaccc058fbb13a0762b5292c525b